src/util/util_log.cpp

```diff
@@ -1,8 +1,26 @@
 #include "util_log.h"
 #include "util_env.h"
+#include "util_string.h"
+
+using PFN_wineDbgOutput = int(__cdecl*)(const char*);
+
+static PFN_wineDbgOutput wineDbgOutput = nullptr;
 
 namespace dxvk::log {
+    void cout(const std::string& message) {
```

Comment:
I think it's kinda weird to name this after console/standard out when in both cases the message ends in console/standard error (or so I believe at least).

Reply:
Yeah, I've struggled with the name.. renamed to `print` now..